NPB3.3-MPI/Dockerfile

```diff
@@ -0,0 +1,53 @@
+#The NBP Benchmark will run on ubuntu
+FROM ubuntu:latest
```

Comment:
Clean the Dockerfile (multiple `apt-get`).
Look at the one provided in parsec-3.0 and use the best practices for `apt-get ...`.